mlx/linalg.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright © 2023 Apple Inc.
+
+#include <iostream>
```

Comment:
Remove unused include

Reply:
Good point. I used this for debugging and forgot to remove.